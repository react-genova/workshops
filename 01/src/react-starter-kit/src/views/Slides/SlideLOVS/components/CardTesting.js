//@flow
import React from 'react';
import { Card, CardElement } from '../../../Card';

const CardTesting = () => (
    <Card title="Testing" description="the better you test, the longer you rest">
        <CardElement title="jest" description="the test engine" url="https://facebook.github.io/jest/"  />
        <CardElement title="react-styleguidist" description="design your components" url="https://react-styleguidist.js.org/"  />
        <CardElement title="enzyme" description="test your components" url="https://github.com/airbnb/enzyme"  />
    </Card>
);

export default CardTesting;