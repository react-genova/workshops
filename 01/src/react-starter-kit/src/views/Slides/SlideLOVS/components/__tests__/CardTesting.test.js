//@flow
import React from 'react';
import { shallow } from 'enzyme';
import CardTesting from '../CardTesting';

describe('Card Testing', () => {

    it('snapshot check', () => {
        const component = shallow(<CardTesting />);
        expect(component).toMatchSnapshot();
    });

});