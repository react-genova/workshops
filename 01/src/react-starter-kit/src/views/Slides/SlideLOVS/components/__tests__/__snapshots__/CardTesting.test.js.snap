// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Testing snapshot check 1`] = `
<Card
  description="the better you test, the longer you rest"
  title="Testing"
>
  <CardElement
    description="the test engine"
    title="jest"
    url="https://facebook.github.io/jest/"
  />
  <CardElement
    description="design your components"
    title="react-styleguidist"
    url="https://react-styleguidist.js.org/"
  />
  <CardElement
    description="test your components"
    title="enzyme"
    url="https://github.com/airbnb/enzyme"
  />
</Card>
`;
